//Test Script
/*
for (Integer i = 1; i <= 100; i++) {
    System.debug(i);
}
*/
PLACEHOLDER
public class CustomException extends Exception {}

try {
    Integer expectedValue = 1;
    
    for (Integer i = 1; i <= 100; i++) {
       
        System.assertEquals(expectedValue, i, 'The number should be ' + expectedValue);
        expectedValue++;
    }

} catch (Exception exc) {
    throw new CustomException('There is an error: ' + exc.getMessage());
}

